third_party/terraform/utils/node_config.go.erb

```diff
@@ -244,6 +244,16 @@ var schemaNodeConfig = &schema.Schema{
 					},
 				},
 			},
+
+			"boot_disk_kms_key": {
+<% if version.nil? || version == 'ga' -%>
```

Comment:
if it's a new field that didn't exist in GA before, we can just add `<% unless version == 'ga' -%>` gates around the field